pkg/controllers/awsloadbalancercontroller/ingressclass.go

```diff
@@ -50,7 +49,7 @@ func (r *AWSLoadBalancerControllerReconciler) ensureIngressClass(ctx context.Con
 	if err != nil && !errors.IsAlreadyExists(err) {
 		return fmt.Errorf("failed to create default IngressClass %s: %w", controller.Spec.IngressClass, err)
 	}
-	return r.updateStatusIngressClass(ctx, controller, controller.Spec.IngressClass)
```

Comment:
Doesn't this ignore changes in `ingressclass.spec.parameters`?

Reply:
Never mind, looks like we aren't supporting it right now. We should include this in the EP that to make it clear that [IngressClassParams](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/ingress/ingress_class/#ingressclassparams) isn't support in the first release.

Reply:
My mistake I thought `updateStatusIngressClass()` was `updateIngressClass()` and it was overwriting `spec.parameters`.